src/components/v2/Layout/Header/styles.ts

```diff
@@ -23,11 +23,6 @@ export const useStyles = () => {
         padding: ${theme.spacing(6, 4, 0)} !important;
       }
     `,
-    rightItemContainer: css`
```

Comment:
I'm curious why this query didn't work.

Reply:
It hides the SVG icon of the logo, which also impacts the icon of the XVS token. We had this issue before